src/re2parser.cc

```diff
@@ -562,16 +562,23 @@ namespace {
  * @param epsilon_value value, that will represent epsilon on transitions
  * @return mata::Nfa::Nfa corresponding to pattern
  */
-void Mata::RE2Parser::create_nfa(Nfa::Nfa* nfa, const std::string& pattern, bool use_epsilon, int epsilon_value) {
+void Mata::RE2Parser::create_nfa(Nfa::Nfa* nfa, const std::string& pattern, bool use_epsilon, int epsilon_value, bool use_reduce) {
```

Comment:
Parameter `use_reduce` could benefit from a documentation string explaining that is uses reduce on the result, as well as trimming the result beforehand. This is just an idea, totally optional, though.

Reply:
ok